.github/workflows/ci.yml

```diff
@@ -1,102 +1,72 @@
-
-
 name: Delta Acceptance Testing
 
 on:
-  pull_request:
   push:
-    branches:
-      - main
-
-env:
-      PYTHON_VERSION: 3.9.13
-
+    branches: 
+      -main
+      -fix-ci
+  pull_request:
 
 jobs:
-
-  analyze:
-    name: Lint Code
+  test:
     runs-on: ubuntu-latest
-  
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-        with:
-          token: ${{ secrets.PAT }}
-
-      - name: Install poetry
-        run: pipx install poetry
+      - uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
+      # If you wanted to use multiple Python versions, you'd have specify a matrix in the job and
+      # reference the matrixe python version here.
+      - uses: actions/setup-python@v2
         with:
-          python-version: ${{ env.PYTHON_VERSION }}
-          cache: 'poetry'          
-
-      - name: Install dependencies and project in dev mode
-        run: |
-          make install
-
-      - name: Lint
-        run: make lint-base lint-mypy
```

Comment:
Are we intentionally not linting?

Reply:
OK, so I think I understand what we're doing now.  We're autoformating the code with autopep8 and lining the code with mypy, flake8, and bandit.  The command is currently called [autolinting](https://github.com/delta-incubator/dat/blob/master/Makefile#L10), but think we mean autoformatting.  Do we need to run the code through three different linters?  Perhaps we can autoformat with black and lint with black so we're just using one tool?

Reply:
@wjones127 - jfyi, the current CI currently errors out with a ["lint code" error](https://github.com/delta-incubator/dat/issues/20#issue-1420771213), but I'm not sure what's the root cause.  Might be good to get the CI in a working state and then add linting back later.